webapi/src/main/scala/org/knora/webapi/util/TransformData.scala

```diff
@@ -33,6 +34,96 @@ import scala.collection.immutable.TreeMap
+    val header6 = Value("h6")
+    // <h6>...</h6>
+    val superscript = Value("sup")
+    // <sup>...</sup>
+    val subscript = Value("sub")
+    // <sub>...</sub>
+    val orderedList = Value("ol")
+    // <ol>...</ol>
+    val unorderedList = Value("ul")
+    // <ul>...</ul>
+    val listElement = Value("li")
+    // <li>...</li>
+    val styleElement = Value("style") // <span>...</span>
+
+    val valueMap: Map[String, Value] = values.map(v => (v.toString, v)).toMap
+
+    /**
+      * Given the name of a value in this enumeration, returns the value. If the value is not found, the provided error function is called.
+      *
+      * @param name     the name of the value.
+      * @param errorFun the function to be called in case of an error.
+      * @return the requested value.
+      */
+    def lookup(name: String, errorFun: () => Nothing): Value = {
+        valueMap.get(name) match {
+            case Some(value) => value
+            case None => errorFun()
+        }
+    }
+
+    /**
+      * Maps standoff tag IRIs to this enumeration's values.
+      */
+    val IriToEnumValue: Map[IRI, LegacyTextattrV1.Value] = new ErrorHandlingMap(Map(
+        OntologyConstants.Standoff.StandoffParagraphTag -> paragraph,
+        OntologyConstants.Standoff.StandoffItalicTag -> italic,
+        OntologyConstants.Standoff.StandoffBoldTag -> bold,
+        OntologyConstants.Standoff.StandoffUnderlineTag -> underline,
+        OntologyConstants.Standoff.StandoffStrikethroughTag -> strikethrough,
+        OntologyConstants.KnoraBase.StandoffLinkTag -> link,
+        OntologyConstants.KnoraBase.StandoffUriTag -> link,
+        OntologyConstants.Standoff.StandoffHeader1Tag -> header1,
+        OntologyConstants.Standoff.StandoffHeader2Tag -> header2,
+        OntologyConstants.Standoff.StandoffHeader3Tag -> header3,
+        OntologyConstants.Standoff.StandoffHeader4Tag -> header4,
+        OntologyConstants.Standoff.StandoffHeader5Tag -> header5,
+        OntologyConstants.Standoff.StandoffHeader6Tag -> header6,
+        OntologyConstants.Standoff.StandoffSuperscriptTag -> superscript,
+        OntologyConstants.Standoff.StandoffSubscriptTag -> subscript,
+        OntologyConstants.Standoff.StandoffOrderedListTag -> orderedList,
+        OntologyConstants.Standoff.StandoffUnorderedListTag -> unorderedList,
+        OntologyConstants.Standoff.StandoffListElementTag -> listElement,
+        OntologyConstants.Standoff.StandoffStyleElementTag -> styleElement
+    ), { key => throw InconsistentTriplestoreDataException(s"Invalid standoff tag IRI: $key") })
+
+
+    // Maps this enumeration's values to standoff tag IRIs.
+
+    val EnumValueToIri: Map[LegacyTextattrV1.Value, IRI] = new ErrorHandlingMap(IriToEnumValue.map(_.swap), { key => throw InconsistentTriplestoreDataException(s"Invalid standoff tag name: $key") })
+}
+
+/**
   * Updates the structure of Knora repository data to accommodate changes in Knora.
   */
 object TransformData extends App {
```

Comment:
Should this add:

* A reference to the default mapping, if a text value doesn't specify one
* Standoff root tags if they're missing